f2/src/forms/LocationInfoForm.js

```diff
@@ -3,53 +3,21 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { jsx } from '@emotion/core'
 import { Trans } from '@lingui/macro'
-import { Form } from 'react-final-form'
-import { NextAndCancelButtons } from '../components/next-and-cancel-buttons'
-import { Stack } from '@chakra-ui/core'
 import { useStateValue } from '../utils/state'
-import { Link as ReactRouterLink } from 'react-router-dom'
-import { Flex, Icon } from '@chakra-ui/core'
+import { Form, Container, Row } from 'react-bootstrap'
+import { Formik, Field, ErrorMessage } from 'formik'
+import { NextCancelButtons, SkipButton } from '../components/formik/button'
+import { LocationInfoFormSchema } from './LocationInfoFormSchema'
 import { P } from '../components/paragraph'
```

Comment:
Should use the <P> component from f2\src\components\formik\paragraph\index.js

Reply:
Thank you for catching it. fixed.